config/locales/en/helpers.yml

```diff
@@ -365,4 +367,110 @@ en:
       travel_expenses_excl_car: Travel expenses (excl car)
       veterinarian_report: Veterinarian Report
       voice_recognition_experts: Voice Recognition
-
+    quote_services:
+      ae_consultant: A&E Consultant
+      accident_reconstruction: Accident Reconstruction
+      accountant: Accountant
+      anaesthatist: Anaesthatist
+      animal_behaviourist: Animal Behaviourist
+      architect: Architect
+      back_calculations: Back Calculations
+      benefit_expert: Benefit Expert
+      culture_expert: Culture expert
```

Comment:
There are some casing inconsistencies here - Culture expert doesn't have a capital E but Benefit Expert does. Is there an official list available? I suspect that the source you're working from was hand-transcribed at some point, as a few things like this have crept in.

Reply:
the list comes from here: https://docs.google.com/spreadsheets/d/1xtepqupYvQpb79CgCyDE-8G90Ol4xvXVtKb1gh08f0A/edit#gid=0

I'll talk to Aimee to check if there's a cleaned up official version